topics/assembly/tutorials/general-introduction/tours/tour.yaml

```diff
@@ -0,0 +1,281 @@
+﻿id: assembly-general-intro
+name: Galaxy Tour
+description: >-
+  In this tour we will perform a de novo assembly of a short read set using the
```

Comment:
Maybe point out that this is a basic de novo assembly for illustrative purposes. The second of the two tutorials offers a more complete view after this one has been completed.